ext/nnstreamer/tensor_filter/tensor_filter_edgetpu.cc

```diff
@@ -295,13 +322,13 @@ edgetpu_subplugin::BuildEdgeTpuInterpreter(
   std::unique_ptr<tflite::Interpreter> interpreter;
   if (tflite::InterpreterBuilder(model, resolver)(&interpreter) !=
       kTfLiteOk) {
-    std::cerr << "Failed to build interpreter." << std::endl;
+    nns_loge ("Failed to build interpreter.");
```

Comment:
Q. Is it okay to keep going on the execution even after this error?

Reply:
A. Yes. Since this method is called multiple times during the negotiation procedure, we cannot stop the pipeline at this moment. Moreover, even if it is failed to build an interpreter, using the interpreter instance is safe.

Reply:
Yes, thanks for the explanation.

But, according to https://github.com/tensorflow/tensorflow/blob/r1.13/tensorflow/lite/model.cc,
this builder can return with uninitialized interpreter in some error cases (e.g, unmatched model schema version).

Anyway, my concern is not related to this PR.

Reply:
@dongju-chae I will take a look into your concern. Thanks.